src/en/discover/case-studies/ceph-for-openstack/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Ceph done right for OpenStack"
+image: "/assets/bitmaps/illustration-openstack-softiron.jpg"
```

Comment:
```suggestion
image: "/assets/bitmaps/illustration-openstack-softiron.png"
```

@fishoutoforder Quick one here. Just a correction on the file extension. Should be `png`